rasa/core/tracker_store.py

```diff
@@ -1325,7 +1325,7 @@ def _event_query(
                 )
             )
 
-        return event_query.order_by(self.SQLEvent.timestamp)
+        return event_query.order_by(self.SQLEvent.timestamp, self.SQLEvent.id)
```

Comment:
Why is the change to order by both? We do not need batch ordering, but simple overall ordering. The SQLEvent.id is unique and chronologically ordered by nature, which makes timestamp redundant.